pkg/render/funcs.go

```diff
@@ -28,3 +32,9 @@ func isSet(m map[string]interface{}, key string) interface{} {
 	}
 	return val
 }
+
+// escapeCharacters returns the given string with any
+// possible reference of '$' escaped.
+func escapeCharacters(text string) string {
```

Comment:
I'd make it e.g. `iniEscapeCharacters`, various formats have various special chars.